lib/create_users.py

```diff
@@ -0,0 +1,54 @@
+from passlib.context import CryptContext
+import json
+
+pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
+def get_password_hash(password):
+    """
+    Generate the hash of a password.
+
+    Parameters:
+    - password (str): The password to be hashed.
+
+    Returns:
+    - str: The hashed password.
+    """
+    return pwd_context.hash(password)
+
+def create_user(db,user_data: dict[str, str]) -> None:

```

Comment:
use dataclassess for this type of implementations. Dictionaries are difficult to maintain in the long term. You can not tell easily the indexes if you do not go directly to the creation.